### PR TITLE
Add guide from telecablesat.fr

### PR DESCRIPTION
**Pull request by freearhey**
#423

### CODE REVIEW

File: sites/telecablesat.fr/telecablesat.fr_fr.channels.xml [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+    <channel lang="fr" xmltv_id="" site_id="524">Alsace 20</channel>
+    <channel lang="fr" xmltv_id="" site_id="2320">Altice Studio</channel>
+    <channel lang="fr" xmltv_id="" site_id="12">Animaux</channel>
+    <channel lang="fr" xmltv_id="" site_id="436">Antenne Réunion</channel>
+    <channel lang="fr" xmltv_id="" site_id="111">Arte</channel>
+    <channel lang="fr" xmltv_id="" site_id="1140">ATV Guadeloupe</channel>
+    <channel lang="fr" xmltv_id="" site_id="1531">ATV Guyane</channel>
+    <channel lang="fr" xmltv_id="" site_id="295">ATV Martinique</channel>
+    <channel lang="fr" xmltv_id="" site_id="15">Automoto</channel>
+    <channel lang="fr" xmltv_id="" site_id="23">één</channel>
```
Comment (LaneSh4d0w) [inline sites/telecablesat.fr/telecablesat.fr_fr.channels.xml:25]:
```suggestion
    <channel lang="be" xmltv_id="" site_id="23">één</channel>

```

File: sites/telecablesat.fr/telecablesat.fr_fr.channels.xml [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+    <channel lang="fr" xmltv_id="" site_id="283">Ciné+ Emotion</channel>
+    <channel lang="fr" xmltv_id="" site_id="401">Ciné+ Famiz</channel>
+    <channel lang="fr" xmltv_id="" site_id="284">Ciné+ Frisson</channel>
+    <channel lang="fr" xmltv_id="" site_id="317">Ciné+ Frisson Belgique</channel>
+    <channel lang="fr" xmltv_id="" site_id="282">Ciné+ Premier</channel>
+    <channel lang="fr" xmltv_id="" site_id="294">Ciné+ Premier Belgique</channel>
+    <channel lang="fr" xmltv_id="" site_id="2781">Clique TV</channel>
+    <channel lang="fr" xmltv_id="" site_id="50">Club RTL</channel>
+    <channel lang="fr" xmltv_id="" site_id="226">CNEWS</channel>
+    <channel lang="fr" xmltv_id="" site_id="53">CNN</channel>
```
Comment (LaneSh4d0w) [inline sites/telecablesat.fr/telecablesat.fr_fr.channels.xml:81]:
```suggestion
    <channel lang="en" xmltv_id="" site_id="53">CNN</channel>
```
English

File: sites/telecablesat.fr/telecablesat.fr_fr.channels.xml [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+    <channel lang="fr" xmltv_id="" site_id="2320">Altice Studio</channel>
+    <channel lang="fr" xmltv_id="" site_id="12">Animaux</channel>
+    <channel lang="fr" xmltv_id="" site_id="436">Antenne Réunion</channel>
+    <channel lang="fr" xmltv_id="" site_id="111">Arte</channel>
+    <channel lang="fr" xmltv_id="" site_id="1140">ATV Guadeloupe</channel>
+    <channel lang="fr" xmltv_id="" site_id="1531">ATV Guyane</channel>
+    <channel lang="fr" xmltv_id="" site_id="295">ATV Martinique</channel>
+    <channel lang="fr" xmltv_id="" site_id="15">Automoto</channel>
+    <channel lang="fr" xmltv_id="" site_id="23">één</channel>
+    <channel lang="fr" xmltv_id="" site_id="16">BBC 1</channel>
```
Comment (LaneSh4d0w) [inline sites/telecablesat.fr/telecablesat.fr_fr.channels.xml:26]:
```suggestion
    <channel lang="en" xmltv_id="" site_id="16">BBC 1</channel>
```

File: sites/telecablesat.fr/telecablesat.fr_fr.channels.xml [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+    <channel lang="fr" xmltv_id="" site_id="17">BBC 2</channel>
+    <channel lang="fr" xmltv_id="" site_id="18">BBC Entertainment</channel>
```
Comment (LaneSh4d0w) [inline sites/telecablesat.fr/telecablesat.fr_fr.channels.xml:28]:
```suggestion
    <channel lang="en" xmltv_id="" site_id="18">BBC Entertainment</channel>

```

File: sites/telecablesat.fr/telecablesat.fr_fr.channels.xml [hunk 5]
```diff
@@ -0,0 +1,288 @@
...
+    <channel lang="fr" xmltv_id="" site_id="19">BBC World News</channel>
```
Comment (LaneSh4d0w) [inline sites/telecablesat.fr/telecablesat.fr_fr.channels.xml:29]:
```suggestion
    <channel lang="en" xmltv_id="" site_id="19">BBC World News</channel>
```

File: sites/telecablesat.fr/telecablesat.fr_fr.channels.xml [hunk 6]
```diff
@@ -0,0 +1,288 @@
...
+    <channel lang="fr" xmltv_id="" site_id="57">Demain TV</channel>
+    <channel lang="fr" xmltv_id="" site_id="400">Discovery Channel</channel>
+    <channel lang="fr" xmltv_id="" site_id="2324">Discovery Family</channel>
+    <channel lang="fr" xmltv_id="" site_id="2184">Discovery Investigation</channel>
+    <channel lang="fr" xmltv_id="" site_id="1374">Discovery Science</channel>
+    <channel lang="fr" xmltv_id="" site_id="58">Disney Channel</channel>
+    <channel lang="fr" xmltv_id="" site_id="1967">Disney Channel Wallonia</channel>
+    <channel lang="fr" xmltv_id="" site_id="300">Disney Junior</channel>
+    <channel lang="fr" xmltv_id="" site_id="560">Dorcel TV</channel>
+    <channel lang="fr" xmltv_id="" site_id="61">DW (English)</channel>
```
Comment (LaneSh4d0w) [inline sites/telecablesat.fr/telecablesat.fr_fr.channels.xml:96]:
```suggestion
    <channel lang="en" xmltv_id="" site_id="61">DW (English)</channel>
```
English

File: sites/telecablesat.fr/telecablesat.fr_fr.channels.xml [hunk 7]
```diff
@@ -0,0 +1,288 @@
...
+    <channel lang="fr" xmltv_id="" site_id="1295">Golf+</channel>
+    <channel lang="fr" xmltv_id="" site_id="329">Guadeloupe la 1ère</channel>
+    <channel lang="fr" xmltv_id="" site_id="482">Gulli</channel>
+    <channel lang="fr" xmltv_id="" site_id="260">Guyane la 1ère</channel>
+    <channel lang="fr" xmltv_id="" site_id="88">Histoire TV</channel>
+    <channel lang="fr" xmltv_id="" site_id="416">Hustler TV</channel>
+    <channel lang="fr" xmltv_id="" site_id="781">I24news</channel>
+    <channel lang="fr" xmltv_id="" site_id="94">Infosport+</channel>
+    <channel lang="fr" xmltv_id="" site_id="1585">J-One</channel>
+    <channel lang="fr" xmltv_id="" site_id="1280">Ketnet</channel>
```
Comment (LaneSh4d0w) [inline sites/telecablesat.fr/telecablesat.fr_fr.channels.xml:147]:
```suggestion
    <channel lang="nl" xmltv_id="" site_id="1280">Ketnet</channel>
```
Flemish Belgium

File: sites/telecablesat.fr/telecablesat.fr_fr.channels.xml [hunk 8]
```diff
@@ -0,0 +1,288 @@
...
+    <channel lang="fr" xmltv_id="" site_id="2037">Crime District</channel>
+    <channel lang="fr" xmltv_id="" site_id="458">CSTAR</channel>
+    <channel lang="fr" xmltv_id="" site_id="13">Das Erste</channel>
+    <channel lang="fr" xmltv_id="" site_id="57">Demain TV</channel>
+    <channel lang="fr" xmltv_id="" site_id="400">Discovery Channel</channel>
+    <channel lang="fr" xmltv_id="" site_id="2324">Discovery Family</channel>
+    <channel lang="fr" xmltv_id="" site_id="2184">Discovery Investigation</channel>
+    <channel lang="fr" xmltv_id="" site_id="1374">Discovery Science</channel>
+    <channel lang="fr" xmltv_id="" site_id="58">Disney Channel</channel>
+    <channel lang="fr" xmltv_id="" site_id="1967">Disney Channel Wallonia</channel>
```
Comment (LaneSh4d0w) [inline sites/telecablesat.fr/telecablesat.fr_fr.channels.xml:93]:
```suggestion
    <channel lang="fr;nl" xmltv_id="" site_id="1967">Disney Channel Wallonia</channel>
```
Both Flemish Dutch and French

File: sites/telecablesat.fr/telecablesat.fr_fr.channels.xml [hunk 9]
```diff
@@ -0,0 +1,288 @@
...
+    <channel lang="fr" xmltv_id="" site_id="1280">Ketnet</channel>
+    <channel lang="fr" xmltv_id="" site_id="110">KTO</channel>
+    <channel lang="fr" xmltv_id="" site_id="1401">L&apos;Equipe</channel>
+    <channel lang="fr" xmltv_id="" site_id="234">La Chaîne parlementaire</channel>
+    <channel lang="fr" xmltv_id="" site_id="892">La Trois</channel>
+    <channel lang="fr" xmltv_id="" site_id="164">La Une</channel>
+    <channel lang="fr" xmltv_id="" site_id="112">LCI - La Chaîne Info</channel>
+    <channel lang="fr" xmltv_id="" site_id="535">LM TV Sarthe</channel>
+    <channel lang="fr" xmltv_id="" site_id="118">M6</channel>
+    <channel lang="fr" xmltv_id="" site_id="683">Man-X</channel>
```
Comment (LaneSh4d0w) [inline sites/telecablesat.fr/telecablesat.fr_fr.channels.xml:156]:
```suggestion
    <channel lang="fr,en" xmltv_id="" site_id="683">Man-X</channel>
```
European-scaled gay adult channel. Cannot confirm if French only, as the programmation is written in English. I do not have this channel myself, ISP providing me only with channels like Brazzers, Penthouse or Dorcel.

File: sites/telecablesat.fr/telecablesat.fr_fr.channels.xml [hunk 10]
```diff
@@ -0,0 +1,288 @@
...
+    <channel lang="fr" xmltv_id="" site_id="1562">Paramount Channel</channel>
+    <channel lang="fr" xmltv_id="" site_id="145">Paris Première</channel>
+    <channel lang="fr" xmltv_id="" site_id="406">Pink TV</channel>
+    <channel lang="fr" xmltv_id="" site_id="344">Piwi+</channel>
+    <channel lang="fr" xmltv_id="" site_id="147">Planète+</channel>
+    <channel lang="fr" xmltv_id="" site_id="402">Planète+ Aventure &amp; Expérience</channel>
+    <channel lang="fr" xmltv_id="" site_id="662">Planète+ Crime Investigation</channel>
+    <channel lang="fr" xmltv_id="" site_id="377">Plug RTL</channel>
+    <channel lang="fr" xmltv_id="" site_id="2326">Polar+</channel>
+    <channel lang="fr" xmltv_id="" site_id="964">Pro7</channel>
```
Comment (LaneSh4d0w) [inline sites/telecablesat.fr/telecablesat.fr_fr.channels.xml:199]:
```suggestion
    <channel lang="de" xmltv_id="" site_id="964">Pro7</channel>
```
ProSieben, german.

File: sites/telecablesat.fr/telecablesat.fr_fr.channels.xml [hunk 11]
```diff
@@ -0,0 +1,288 @@
...
+    <channel lang="fr" xmltv_id="" site_id="1075">Proximus 11</channel>
+    <channel lang="fr" xmltv_id="" site_id="154">Rai Due</channel>
```
Comment (LaneSh4d0w) [inline sites/telecablesat.fr/telecablesat.fr_fr.channels.xml:201]:
```suggestion
    <channel lang="it" xmltv_id="" site_id="154">Rai Due</channel>
```
Italian

File: sites/telecablesat.fr/telecablesat.fr_fr.channels.xml [hunk 12]
```diff
@@ -0,0 +1,288 @@
...
+    <channel lang="fr" xmltv_id="" site_id="155">Rai Tre</channel>
```
Comment (LaneSh4d0w) [inline sites/telecablesat.fr/telecablesat.fr_fr.channels.xml:202]:
```suggestion
    <channel lang="it" xmltv_id="" site_id="155">Rai Tre</channel>
```

File: sites/telecablesat.fr/telecablesat.fr_fr.channels.xml [hunk 13]
```diff
@@ -0,0 +1,288 @@
...
+    <channel lang="fr" xmltv_id="" site_id="156">Rai Uno</channel>
```
Comment (LaneSh4d0w) [inline sites/telecablesat.fr/telecablesat.fr_fr.channels.xml:203]:
```suggestion
    <channel lang="it" xmltv_id="" site_id="156">Rai Uno</channel>
```

File: sites/telecablesat.fr/telecablesat.fr_fr.channels.xml [hunk 14]
```diff
@@ -0,0 +1,288 @@
...
+    <channel lang="fr" xmltv_id="" site_id="704">Vià Occitanie Montpellier</channel>
+    <channel lang="fr" xmltv_id="" site_id="538">Vià Occitanie Pays Gardois</channel>
+    <channel lang="fr" xmltv_id="" site_id="1045">ViàMoselleTV</channel>
+    <channel lang="fr" xmltv_id="" site_id="1095">ViàVosges</channel>
+    <channel lang="fr" xmltv_id="" site_id="2171">Vice TV</channel>
+    <channel lang="fr" xmltv_id="" site_id="1144">Voo Foot</channel>
+    <channel lang="fr" xmltv_id="" site_id="413">VOOsport World 1</channel>
+    <channel lang="fr" xmltv_id="" site_id="414">VOOsport World 2</channel>
+    <channel lang="fr" xmltv_id="" site_id="472">VOOsport World 3</channel>
+    <channel lang="fr" xmltv_id="" site_id="215">VTM</channel>
```
Comment (LaneSh4d0w) [inline sites/telecablesat.fr/telecablesat.fr_fr.channels.xml:280]:
```suggestion
    <channel lang="nl" xmltv_id="" site_id="215">VTM</channel>
```
Flemish Dutch

File: sites/telecablesat.fr/telecablesat.fr_fr.channels.xml [hunk 15]
```diff
@@ -0,0 +1,288 @@
...
+    <channel lang="fr" xmltv_id="" site_id="119">W9</channel>
+    <channel lang="fr" xmltv_id="" site_id="519">Wallis et Futuna la 1ère</channel>
+    <channel lang="fr" xmltv_id="" site_id="2334">Warner TV</channel>
+    <channel lang="fr" xmltv_id="" site_id="809">Wéo</channel>
+    <channel lang="fr" xmltv_id="" site_id="218">XXL</channel>
+    <channel lang="fr" xmltv_id="" site_id="219">ZDF</channel>
```
Comment (LaneSh4d0w) [inline sites/telecablesat.fr/telecablesat.fr_fr.channels.xml:286]:
```suggestion
    <channel lang="de" xmltv_id="" site_id="219">ZDF</channel>
```
German

File: sites/telecablesat.fr/telecablesat.fr_fr.channels.xml (hunk 9 above)
Comment (freearhey) [reply sites/telecablesat.fr/telecablesat.fr_fr.channels.xml:156]: Strangely, my program is in French:

image

And in this file you can not specify multiple languages at the same time.

Comment (LaneSh4d0w) [reply sites/telecablesat.fr/telecablesat.fr_fr.channels.xml:156]: Had a less detailed programmation sorry, firewall blocks this website here.

image

File: sites/telecablesat.fr/telecablesat.fr_fr.channels.xml (hunk 8 above)
Comment (freearhey) [reply sites/telecablesat.fr/telecablesat.fr_fr.channels.xml:93]: This attribute specifies only the language of the program, not the broadcast, and in this case it is French:

image

Comment (LaneSh4d0w) [reply sites/telecablesat.fr/telecablesat.fr_fr.channels.xml:93]: https://en.wikipedia.org/wiki/Disney_Channel_(Belgian_TV_channel)

Comment (freearhey) [reply sites/telecablesat.fr/telecablesat.fr_fr.channels.xml]: I meant the guide.

**Pull request merged**